cl/scrapers/management/commands/cl_scrape_opinions.py

```diff
@@ -346,7 +358,7 @@ def scrape_court(self, site, full_crawl=False, ocr_available=True):
             # Only update the hash if no errors occurred.
             dup_checker.update_site_hash(site.hash)
 
-    def parse_and_scrape_site(self, mod, full_crawl):
+    def parse_and_scrape_site(self, mod, full_crawl, _, __):
```

Comment:
Here pass the options instead of adding the params from the backscraper

```
    def parse_and_scrape_site(self, mod, options):
        site = mod.Site().parse()
        self.scrape_court(site, options["full_crawl"])
```